.github/workflows/ethereum-ts.yml

```diff
@@ -1,57 +0,0 @@
-name: Ethereum Typescript
-on:
-  push:
-    paths:
-      - 'ethereum/gateway-eth-ts/**'
-      - '.github/**'
-  pull_request:
```

Comment:
Removed job to run test on the current ts-client for gateway. The contract changes being made break the current gateway client test.